### PR TITLE
Enhancement/UI themes

### PR DESCRIPTION
**Pull request by 46chris**
What's in the PR: 
- Changed a lot of the logic for TomatoSlider.js, and TimerUI.js. Svg's are being dynamically loaded in so all static requests to obtain elements of the svg had to be changed. Changed functions / implemented custom events / made attributechange observers such that the functionality of the component operates after the svg's are loaded in 
- Changed the jest unit tests for TimerUI.js to be suitable for the new logic 
- Added in a moduleNameMapper / mocks folder so imports for static media doesn't throw an error with jest
- Added Icon change setting / functionality according to how others did it  
- Added IntroIcons.js component to change the icons on the home page according to the icon setting 
- Added in Bomb icons
- Changed the existing svg files to make them compatible with the new logic and styled accordingly to main.scss

Couple notes: 
- The page reloads if the icon setting is changed so the svg's can rerender with the new Data URL. That's just the easiest way that came to mind, feel free to suggest anyway to do that without a page reload if anything comes to mind 
- The icon on timerUI renders in a new svg when it switches between pomo and break sessions. I did that because the break icon I made for the bomb icon has svg paths different from the original so I couldn't really use class change to change between the two (could be done with animations I think but that's probably going to take a minute). Regardless, that's all to say that the icon on the timer page disappears and reappears with the new icon between sessions instead of a smooth class change. If the stutter is too distracting or egregious I can work on changing the icon or looking into animations so class changes work. 
- Adding tasks on the create session page is nowhere smooth as it was before I think there's probably a way to speed this up. We could probably make that another task, I don't think it would take too long. 
- Idk how bug-free the implementation is, it seems alright on general usage but I wouldn't be surprised if some of the code doesn't work.

### CODE REVIEW

**Comment by xingyuzzzhu** [comment]
Keyboard shortcut doesn't work on the Task List page when adding a task.

File: source/js/components/TimerUI.js [hunk 1]
```diff
@@ -109,6 +144,60 @@ class TimerUI extends HTMLElement {
   setColorGold() {
     this.querySelector('.timer-image').classList.add('gold-tomato');
   }
+
+  render() {
+    const timerSVG = this.querySelector('#timerIcon');
+    timerSVG.addEventListener('load', () => {
+      let svgDoc = timerSVG.contentDocument;
+
+      // Fuck you jest
```
Comment (HarrisSteven) [inline source/js/components/TimerUI.js:153]: amen

Comment (markbussard) [reply source/js/components/TimerUI.js:153]: 😂😂

File: source/js/components/TomatoSlider.js [hunk 2]
```diff
@@ -14,11 +18,24 @@ class TomatoSlider extends HTMLElement {
...
   constructor() {
     super();
-    this.appendChild(document.querySelector('#tomato-slider-template').content.cloneNode(true));
+
+    this.svgUrls = svgIcons[backend.get('Icon')].urls;
+    this.svgClasses = svgIcons[backend.get('Icon')].classes;
+    // Create template and append to tomato-slider (need to maintain input child)
+    const template = document.createElement('template');
+    template.innerHTML = `
+      <div class="d-flex h-100 justify-content-between align-items-center slider-tomato-container">
```
Comment (markbussard) [inline source/js/components/TomatoSlider.js:27]: So, I believe the 'h-100' class is what causes the rows to load drastically in height before all the icons are re rendered. If we could adjust the styling so the row heights remain consistent during icons loading and is not noticeable to user.

Comment (46chris) [reply source/js/components/TomatoSlider.js]: Fix done by Mark

**Review by markbussard — APPROVED** [review]

**Pull request merged**